### ISSUE TITLE
Percentage of dataset division

### ISSUE THREAD

**Issue by Lxycherryup** [YChienHung/PolypMix#2]
Hi, can you describe the division ratio of your training set, validation set, and test set?

**Comment by YChienHung** [comment]
We first sort the dataset according to the file name and save it in the list, then use the random number seed 8888 to shuffle the list, and then divide it according to the ratio of [0.6, 0.2, 0.2], training set, validation set and test set.
